src/main.rs

```diff
@@ -391,7 +373,7 @@ fn main() {
         println!(); // print a newline
     }
     let format;
-
+    
```

Comment:
Whitespace. You should run `cargo fmt`.

Reply:
We can do that in sep branch after this pr is merged

Reply:
I added in that white space. I have a habit of adding extra white space to make the code easier to read. Is that a bad practice?

Reply:
You added four spaces to an otherwise empty line, I don't think that was intentional. You can see it in the darker green area.